File: Application/Application/TableValues/TableValue.cs
using System;
using Microsoft.Azure.Cosmos.Table;

namespace Application.TableValues
{
    public class TableValue : TableEntity
    {
        public TableValue() { }

        public TableValue(string id, string deviceId)
        {
            PartitionKey = deviceId;
            RowKey = id;
        }

        public Double? Humidity { get; set; }
        public Double? Pressure { get; set; }
        public Double? Temperature { get; set; }
        public DateTime? SentTimestamp { get; set; }
        
    }
}

File: Application/client-app/src/app/modules/TableValue.ts
export interface ITableValuesEnvelope {
  tableValues: ITableValue[];
  valuesCount: number;
  continuationToken: IContinuationToken;
}

export interface ITableValue {
  partitionKey: string;
  rowKey: string;
  humidity: number;
  pressure: number;
  temperature: number;
  sentTimestamp: Date;
  timestamp: Date;
  etag: string;
}

export interface IContinuationToken {
  nextPartitionKey: string | null;
  nextRowKey: string | null;
  nextTableName: string | null;
  targetLocation: number;
}
